CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # CHANGELOG
 
+## 4.1.5
+
+* Respect the defaultConfigFile and defaultDatabaseDirectory variables in
+  the main package again. They were ignored in 4.1.0 through 4.1.4. If not
+  specified, these GitHub and PPA releases looked for the config in
```

Comment:
"these" => "the"?

Reply:
Also, "looked for .. the database directory in" is a bit awkward.

Reply:
True. I reworded it. Thank you.